covasim/webapp/cova_app.js

```diff
@@ -55,7 +60,11 @@ const interventionTableConfig = {
     },
     symptomatic_testing: {
         formTitle: "Symptomatic testing",
-        fields: [{key: 'start', type: 'number', label: 'Start day'}, {key: 'end', type: 'number', label: 'End day'}, {label: 'Coverage', key: 'level', type: 'select', options: [{label: '10% per day', value: '10'}, {label: '30% per day', value: '30'},]}],
+        fields: [
+            {key: 'start', type: 'number', label: 'Start day', tooltip: 'Start day'},
+            {key: 'end', type: 'number', label: 'End day', tooltip: 'End day'},
+            {key: 'level', type: 'number', label: 'Coverage', tooltip: 'Test coverage', min: 0, max: 100}
```

Comment:
Suggest making the tooltip "Test coverage per day".

Reply:
Is it a percentage? How about "Percent of population covered per day" or something more specific

Reply:
@mfisher-idmod @MIzzo-IDM working on it :)